.github/workflows/fast-forward.yml

```diff
@@ -0,0 +1,36 @@
+name: Fast forward merge
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  fast-forward:
+    if: >-
+      ${{
+        github.event.label.name == 'fast-forward'
+        && contains(fromJson('["release", "release-proxy", "release-compute"]'), github.base_ref)
+      }}
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Remove fast-forward label to PR
+        env:
+          GH_TOKEN: ${{ secrets.CI_ACCESS_TOKEN }}
+        run: |
+          gh pr edit ${{ github.event.pull_request.number }} --repo "${GITHUB_REPOSITORY}" --remove-label "fast-forward"
+
+      - name: Fast forwarding
+        uses: sequoia-pgp/fast-forward@ea7628bedcb0b0b96e94383ada458d812fca4979
+        # See https://docs.github.com/en/graphql/reference/enums#mergestatestatus
+        if: ${{ github.event.pull_request.mergeable_state  == 'clean' }}
+        with:
+          merge: true
+          comment: on-error
+          github_token: ${{ secrets.CI_ACCESS_TOKEN }}
+
+      - name: Comment if mergeable_state is not clean
+        if: ${{ github.event.pull_request.mergeable_state  != 'clean' }}
+        run: |
+          gh pr comment ${{ github.event.pull_request.number }} \
+            --repo "${GITHUB_REPOSITORY}" \
+            --body "Not trying to forward pull-request, because \`mergeable_state\` is \`${{ github.event.pull_request.mergeable_state }}\`, not \`clean\`."
```

Comment:
I would also add actions that have to be done to get into this state. 
Resolve merge conflicts? Rebase the branch?

Reply:
Well that depends on the exact state. If the PR has been created with the workflow in the repo, this *always* means CI hasn't passed (yet?).

With `-s ours` in the merge, we're ignoring any conflicts. Rebasing is not necessary either, because we've created a fresh merge commit that has direct parents on the source branch and the release branch, meaning we'll always be able to fast-forward.

I can have different comments for each `mergeable_state` to explain what they mean if you'd prefer that?

Reply:
Ah, so basically the "Merge" button in the GitHub UI should be available/green?
Then I'd just say something like that.

Reply:
Yep, that should be the case. I'd assume people will usually merge with the button anyway.